### ISSUE TITLE
open Link in a new tab

### ISSUE THREAD

**Issue by volfclub** [alex-shpak/hugo-book#362]
If it is an external link, open the webpage in a new tab; click on the internal link, and it will be in the current tab.

```
<a href="{{ .Destination | safeURL }}"{{ with .Title}} title="{{ . }}"{{ end }}{{ if strings.HasPrefix .Destination "http" }} target="_blank" rel="noopener"{{ end }}>{{ .Text | safeHTML }}</a>
```

https://discourse.gohugo.io/t/simple-way-to-open-in-a-new-tab/28677/5

This is more useful in web pages with more external links.

**Comment by alex-shpak** [comment]
Hi!
You can add thi to your website if you want, by creating `layouts/_default/_markup/render-link.html` in your websire sources.

**Comment by volfclub** [comment]
> 
> 
> Hi!
> You can add thi to your website if you want, by creating `layouts/_default/_markup/render-link.html` in your websire sources.

🤨 It's a little weird, this doesn't work on the home page（ https://awesome-program-files.vercel.app/ ）; but the other document pages work（ https://awesome-program-files.vercel.app/docs/example/table-of-contents/ ）.

**Comment by volfclub** [comment]
>  The difference is that on your homepage, the links are inside a shortcode ( {{< columns >}} ).
> 
> Inside the shortcode, markdownify is being used. However, from markdownify | Hugo :
> 
>  Note : if you need Render Hooks 1, which markdownify doesn’t currently support, …

https://discourse.gohugo.io/t/how-to-open-link-in-a-new-tab-with-hugo-book/33872/2?u=volfclub

@alex-shpak This is the reason why the homepage does not work.

**Comment by alex-shpak** [comment]
Fixed via #364
